src/components/collection-progress-group/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react'
+import ButtonLink from 'components/button-link'
+import { useCollectionProgress } from 'hooks/progress'
+import { Collection } from 'lib/learn-client/types'
+import { parseCollectionProgress } from './helpers'
+import CollectionProgressStatusSection from './components/collection-progress-status-section'
+import s from './collection-progress-group.module.css'
+
+/**
+ * Displays collection progress status and CTA.
+ */
+function CollectionProgressGroup({ collection }: { collection: Collection }) {
```

Comment:
I've refactored the component formerly known as `CollectionMetaProgress` (as introduced in #989), renaming it to `CollectionProgressGroup` (matches the name in the design spec) and lifting it into `components`.

Rationale is that bits of this component are re-used in the tutorials sidebar content (for the purpose of this PR), and may be re-usable in collection cards as well.